client/src/app/pages/tasks/tasks-page.tsx

```diff
@@ -189,6 +190,11 @@ export const TasksPage: React.FC = () => {
     columnState,
   } = tableControls;
 
+  const tooltips: Record<string, ThInfoType> = {
+    priority: { tooltip: t("tooltip.priority") },
+    preemption: { tooltip: t("tooltip.preemption") },
+  };
```

Comment:
The tooltip keys you added are at `wizard.tooltip.*`

Probably need to create a top level "tooltip" section, or move it to the "message" section.